### PR TITLE
Add settings section to docs

### PR DESCRIPTION
**Pull request by rangelfinal**
Add a settings section to README as suggested on https://github.com/vintasoftware/django-templated-email/pull/131#discussion_r663258534

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -360,6 +360,27 @@ Methods:
...
 
+Settings
+=============
+
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email
```
Comment (fjsj) [inline README.rst:371]: `# The backend that will send the email` -> `# The backend class that will send the email, as a string like 'foo.bar.TemplateBackend' or the class reference itself`

File: README.rst [hunk 2]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email  
+    TEMPLATED_EMAIL_TEMPLATE_DIR    # The directory containing the templates, use '' if using the top level
+    TEMPLATED_EMAIL_FILE_EXTENSION  # The file extension of the template files
+    TEMPLATED_EMAIL_AUTO_PLAIN      # Disable the behavior of calculating the plain part from the html part of the email when `html2text <https://pypi.python.org/pypi/html2text>` is installed
+    TEMPLATED_EMAIL_PLAIN_FUNCTION  # Specify a custom function that converts from HTML to the plain part
```
Comment (fjsj) [inline README.rst:375]: Do we have a default? If so, please specify

File: README.rst [hunk 3]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+=============
+
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email  
+    TEMPLATED_EMAIL_TEMPLATE_DIR    # The directory containing the templates, use '' if using the top level
+    TEMPLATED_EMAIL_FILE_EXTENSION  # The file extension of the template files
```
Comment (fjsj) [inline README.rst:373]: Do we have a default? If so, please specify

File: README.rst [hunk 4]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+Settings
+=============
+
+You can configure Django-Templated-Email by setting the following settings
+
+.. code-block:: python
+
+    TEMPLATED_EMAIL_FROM_EMAIL      # String containing the email to send the email from - fallback to DEFAULT_FROM_EMAIL  
+    TEMPLATED_EMAIL_BACKEND         # The backend that will send the email  
+    TEMPLATED_EMAIL_TEMPLATE_DIR    # The directory containing the templates, use '' if using the top level
```
Comment (fjsj) [inline README.rst:372]: Do we have a default? If so, please specify

File: README.rst (hunk 1 above)
Comment (fjsj) [reply README.rst:371]: Do we have a default? If so, please specify

File: README.rst [hunk 5]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+    TEMPLATED_EMAIL_FILE_EXTENSION  # The file extension of the template files
+    TEMPLATED_EMAIL_AUTO_PLAIN      # Disable the behavior of calculating the plain part from the html part of the email when `html2text <https://pypi.python.org/pypi/html2text>` is installed
```
Comment (fjsj) [inline README.rst:374]: Disable or Enable? True means enable or disable here?

File: README.rst [hunk 6]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+    TEMPLATED_EMAIL_PLAIN_FUNCTION  # Specify a custom function that converts from HTML to the plain part
+
+    # vanilla_django
+    TEMPLATED_EMAIL_DJANGO_SUBJECTS # Deprecated, prefer to use **{% block subject %}**
```
Comment (fjsj) [inline README.rst:378]: You can hide this one from this part of the docs. Remove this line.

File: README.rst [hunk 7]
```diff
@@ -360,6 +360,27 @@ Methods:
...
+
+    # anymail
```
Comment (fjsj) [inline README.rst:380]: # Specific for anymail integration:

**Review by fjsj — CHANGES_REQUESTED** [review]
Thanks! Please check comments

File: README.rst (hunk 1 above)
Comment (rangelfinal) [reply README.rst]: Updated and added default value

File: README.rst (hunk 2 above)
Comment (rangelfinal) [reply README.rst]: Added default value to all settings

File: README.rst (hunk 6 above)
Comment (rangelfinal) [reply README.rst]: Removed

File: README.rst (hunk 7 above)
Comment (rangelfinal) [reply README.rst]: Updated

File: README.rst (hunk 5 above)
Comment (rangelfinal) [reply README.rst]: Updated to display the default value of `True` and updated text to `Set to false to disable the behavior` to make it clearer that `True` enables the behavior and `False` disables it

**Review by fjsj — APPROVED** [review]
Approved. But seems tests have errors? Please check if main suffers from the same issue.

**Review by fjsj — APPROVED** [review]

**Pull request merged**